packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -12,11 +15,18 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
       favicon = pluginOptions.icons[0].src
     }
 
+    if (!cacheId) {
+      cacheId = crypto
```

Comment:
We recently added a `createContentDigest` helper which is exposed to plugins! Would you want to update this PR to use that helper?

```js
exports.onRenderBody = ({ createContentDigest, setHeadComponents }, pluginOptions) => {
  if (!cacheId) {
    cacheId = createContentDigest(fs.readFileSync(`public${favicon}`))
  }
}
```

Additionally, you'll want to add a peerDependency of the version of gatsby that it was introduced in, e.g.

```json
{
  "peerDependencies": {
    "gatsby": ">=2.0.15"
  }
}
```